mxcubeweb/core/components/user/usermanager.py

```diff
@@ -433,37 +450,26 @@ def _login(self, login_id: str, password: str) -> LimsSessionManager:
         if inhouse and not (inhouse and is_local_host()):
             raise Exception("In-house only allowed from localhost")
 
-        # Only allow other users to log-in if they are from the same proposal
-        if (
-            active_users
-            and (login_id not in [p.split("-")[0] for p in active_users])
-            and not HWR.beamline.lims.is_user_login_type()
-        ):
-            raise Exception("Another user is already logged in")
-
-        # Only allow if no one else is logged in
-        if not current_user.is_anonymous:
-            if (
-                active_users
-                and current_user.username != login_id
-                and HWR.beamline.lims.is_user_login_type()
-            ):
-                raise Exception("Another user is already logged in")
-
         # Only allow local login when remote is disabled
         if not self.app.ALLOW_REMOTE and not is_local_host():
             raise Exception("Remote access disabled")
 
         return session_manager
 
-    def _signout(self):
+    def _signout(self, sso_data={}):
```

Comment:
```suggestion
    def _signout(self, sso_data=None):
```

Is it one of those cases where having a mutable thing as default argument might lead to unexpected results...

Examples:
* <https://www.geeksforgeeks.org/use-mutable-default-value-as-an-argument-in-python/>
* <https://nikos7am.com/posts/mutable-default-arguments/>
* <https://www.pullrequest.com/blog/python-pitfalls-the-perils-of-using-lists-and-dicts-as-default-arguments/>

Looking at the code of the method, it seems like it should be fine, but it still feels strange to me to have a dict as default argument.